apiserver/facades/client/client/backend.go

```diff
@@ -189,6 +213,10 @@ func (s stateShim) MongoSession() MongoSession {
 	return MongoSessionShim{s.State.MongoSession()}
 }
 
+type modelShim struct {
+	*state.Model
```

Comment:
It's cool how you can just use an embedded field here, rather than forwarding all the individual methods through -- nice!

Reply:
We are forced to adopt this pattern in lots and lots of places because state methods operate with structs everywhere rather than interfaces those structs cannot be instantiated outside of state and we want to be able to set up unit tests in this package without needing a proper state.State or state.Model etc